Libplanet/Net/BlockDownloadState.cs

```diff
@@ -26,5 +26,10 @@ public class BlockDownloadState : PreloadState
 
         /// <inheritdoc />
         public override int CurrentPhase => 1;
+
+        /// <summary>
+        /// The peer which sent the block.
+        /// </summary>
+        public BoundPeer SourcePeer { get; internal set; }
```

Comment:
What is this used for?

Reply:
I added it to debug easier because sender(peer) can be changed during preload by this pull-request.